src/components/JobManagementTable.tsx

```diff
@@ -0,0 +1,205 @@
+              <TableHead className="w-[200px]">Job Title</TableHead>
+              <TableHead className="w-[150px]">JobType</TableHead>
+              <TableHead className="w-[200px]">Location</TableHead>
+              <TableHead className="w-[200px]">isVerified</TableHead>
+              <TableHead className="text-right">Actions</TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {DispJobs?.map((job) => (
+              <TableRow key={job?.id}>
+                <TableCell className="font-medium">{job?.title}</TableCell>
+                <TableCell>{job?.workMode}</TableCell>
+                <TableCell>{job?.city}</TableCell>
+                <TableCell>
+                  {job.isVerifiedJob ? (
+                    <span className="bg-green-400 p-1 rounded-md text-secondary">
+                      Approved
+                    </span>
+                  ) : (
+                    <ApproveJobDialog
+                      title="Approve this Job?"
+                      description='"Approving this job will make it visible to users"'
+                      handleClick={() => {
+                        handleApproveJob(job.id);
+                      }}
+                    />
+                  )}
+                </TableCell>
+                <TableCell className="text-right w-full flex justify-end">
+                  <span className="mr-5" role="button">
+                    <Edit />
+                  </span>
+                  <span role="button">
+                    <Dialog>
+                      <DialogTrigger>
+                        <DeleteIcon />
+                      </DialogTrigger>
+                      <DialogContent>
+                        <DialogHeader>
+                          <DialogTitle>Are you absolutely sure?</DialogTitle>
+                          <DialogDescription>
+                            This action cannot be undone. This will Delete the
+                            Selected JOB.
+                          </DialogDescription>
+                        </DialogHeader>
+                        <DialogFooter>
+                          <Button
+                            className="mt-2"
+                            variant={'destructive'}
+                            onClick={() => handleDelete(job.id)}
+                          >
+                            Delete
+                          </Button>
+                        </DialogFooter>
+                      </DialogContent>
+                    </Dialog>
+                  </span>
+                </TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+        <Pagination>
+          <PaginationContent>
+            {totalPages ? (
+              <PaginationItem>
+                <PaginationPreviousButton
+                  searchParams={searchParams}
+                  currentPage={currentPage}
+                  baseUrl={APP_PATHS.JOBS}
```

Comment:
can you also remove the need for this to be passed , its not required

Reply:
Pagination not required?

Reply:
no , we don't need the baseUrl to be paaed